ansible/group_vars/all

```diff
@@ -46,27 +55,14 @@ nginx_vhosts:
         auth_basic_user_file /usr/local/biblebox/etc/basicauth;
       }
 
-      location /shared/ {
```

Comment:
The fallback to web root content has been taken away, but I took it out without realising the intent :-\ I think it's nice to have that feature, so let's merge this (so @furnox can continue) and I'll put it back in a followup PR.